tests/contracts/fungible-token/contract.scilla

```diff
@@ -7,6 +7,8 @@ scilla_version 0
 (***************************************************)
 library FungibleToken
 
+let decimals = Uint32 12
```

Comment:
Can you leave `decimal` as an `init` parameter? It's generally a bad idea to leave hard-coded values around.